woocommerce/woocommerce
# Add survey after disabling new experience

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds a survey when the user disables the new product management experience.

![Screenshot 2023-01-20 at 10 48 01](https://user-images.githubusercontent.com/1314156/213712456-a88cccf3-7e14-4c3f-b0a8-4a244c2d6073.png)

It will be visible every time the user disables the new experience.


## UPDATE
🚧🚧🚧🚧🚧 This PR has been split into these two PRs: [36544](https://github.com/woocommerce/woocommerce/pull/36544) and [36532](https://github.com/woocommerce/woocommerce/pull/36532) 🚧🚧🚧

Closes #36324.






### How to test the changes in this Pull Request:



1. Go to a new product using the new product management experience and disable it. Use the bottom bar button or select the "use the classic editor" option at the top bar.

![Screenshot 2023-01-06 at 15 53 57](https://user-images.githubusercontent.com/1314156/211079667-f9f42643-a889-4372-a189-0d258632fdad.png)

<img width="929" alt="Captura de ecrã 2022-10-17, às 16 29 58" src="https://user-images.githubusercontent.com/3380168/196220119-4850d6a3-a0aa-4360-abb8-e6a88ce1142f.png">

2. Make sure you have tracks enabled and also outputted in the console: `localStorage.setItem( 'debug', 'wc-admin:*' );`
3. A survey will be shown, and the `Send feedback` button should be disabled until an option is selected or a text is added to the text area. 
4. Send feedback and verify that the checked options and comment are being tracked.
5. A notice should be shown in the old experience saying:
<img width="599" alt="Screenshot 2023-01-06 at 15 48 59" src="https://user-images.githubusercontent.com/1314156/211078856-1e9b8a11-389f-41a8-af23-5164d3640f53.png">
6. Verify that the `Skip` button and the `X` to close the modal work as expected. 



### Other information:




### FOR PR REVIEWER ONLY:
